config.example.toml

```diff
@@ -84,6 +84,9 @@
 # Wheter to build Enzyme as AutoDiff backend.
 #enzyme = false
 
+# Whether to build LLVM with support for it's gpu offload runtime.
+#offload = false
```

Comment:
Maybe call it gpu-offload? Or does it also support offloading to other hardware like programmable network controllers and flash drives?

Reply:
only compute devices, so no controllers or flash drives, but also non-gpu hardware: 
https://clang.llvm.org/docs/OffloadingDesign.html#openmp-offloading
> supports [..] target offloading to several different architectures such as NVPTX, AMDGPU, X86_64, Arm, and PowerPC.
But then again GPUs are surely the most popular use-case (though afaik there is a hope to support other coprocessors too).
I don't care strongly for either and for some co-processors like TPUs support we probably want MLIR instead of offload anyway, which would be a follow-up project.